corehq/tests/nose.py

```diff
@@ -256,17 +260,33 @@ def iter_names(test):
 
 
 class HqdbContext(DatabaseContext):
-    """Database context with couchdb setup/teardown
+    """Database setup/teardown
 
     In addition to the normal django database setup/teardown, also
-    setup/teardown couchdb databases.
-
-    This is mostly copied from
-    ``couchdbkit.ext.django.testrunner.CouchDbKitTestSuiteRunner``
+    setup/teardown couch databases. Database setup/teardown may be
+    skipped, depending on the presence and value of an environment
+    variable (`REUSE_DB`). Typical usage is `REUSE_DB=1` which means
+    skip database setup and migrations if possible and do not teardown
+    databases after running tests. If connection fails for any test
+    database in `settings.DATABASES` all databases will be re-created
+    and migrated.
+
+    Other supported `REUSE_DB` values:
+
+    - `REUSE_DB=reset` : drop existing, then create and migrate new test
+      databses, but do not teardown after running tests. This is
+      convenient when the existing databases are outdated and need to be
+      rebuilt.
+    - `REUSE_DB=optimize` : same as reset, but use migration optimizer
+      to reduce the number of database migrations.
+    - `REUSE_DB=teardown` : skip database setup; do normal teardown after
+      running tests.
```

Comment:
If `REUSE_DB` isn't set, then I assume this should fall back to the current behavior, right?